src/plugins/intel_gpu/tests/passes/handle_reshape.cpp

```diff
@@ -50,3 +50,42 @@ TEST(handle_reshape, dont_remove_reshape_that_changes_rank) {
 
     ASSERT_TRUE(prog->get_node("reshape").can_be_optimized());
 }
+
```

Comment:
(no need to apply in this PR) Please add a comment: '''// Onednn FC does not support fp32 input, fp16 weight. In such case, we need to ignore reorder_split from handle_reshape pass'''

Reply:
Updated. Thanks!